CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
 ## dbt 0.19.0 (Release TBD)
+- Add select/deselect option in DAG view dropups. ([docs#98](https://github.com/fishtown-analytics/dbt-docs/issues/98))
+
+Contributors:
```

Comment:
can you just add yourself as a contributor in the existing section below? Looks like we have two contributor sections for the 0.19.0 release now :)

Reply:
Ahh, yep just corrected it thanks.